firebase-perf/src/main/java/com/google/firebase/perf/metrics/AppStartTrace.java

```diff
@@ -122,6 +124,13 @@ public static void setLauncherActivityOnResumeTime(String activity) {
     // no-op, for backward compatibility with old version plugin.
   }
 
+  public static Timer getProcessStartTimer() {
```

Comment:
Was this meant to be included in this PR? It looks like it should be in a separate one + it isn't being called or tested.

Reply:
Good catch, removed.